src/herbivoresAndCarnivores.test.js

```diff
@@ -34,8 +34,7 @@ describe('Animal', () => {
 
       expect(carn)
         .toHaveProperty('health', 100);
-    }
-  );
+    });
 
   it(`Herbivore should have 'hide' method which inverts 'hidden' property`,
```

Comment:
The test description for the 'hide' method is incorrect. According to the task, the 'hide' method should set the 'hidden' property to true, not invert it. Please update the test description to reflect the correct behavior.